app/src/main/java/org/sopt/and/MyActivity.kt

```diff
@@ -0,0 +1,194 @@
+package org.sopt.and
+
+import android.os.Bundle
+import android.widget.Space
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            ANDANDROIDTheme {
+                val email = intent.getStringExtra("email")
+                if (email != null) {
+                    myView(email)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun headerProfile(email: String) {
+    Box(
+        modifier = Modifier
+            .background(color = Color.DarkGray)
+            .fillMaxWidth()
+            .padding(15.dp, 15.dp, 15.dp, 15.dp)
```

Comment:
```suggestion
            .padding(15.dp)
```
상하좌우의 패딩 값이 모두 같다면 한 번에 적어주셔도 됩니다.